pvlib/tests/test_irradiance.py

```diff
@@ -281,13 +281,35 @@ def test_sky_diffuse_zenith_close_to_90(model):
     assert sky_diffuse < 100
 
 
-def test_get_sky_diffuse_invalid():
+def test_get_sky_diffuse_model_invalid():
     with pytest.raises(ValueError):
         irradiance.get_sky_diffuse(
             30, 180, 0, 180, 1000, 1100, 100, dni_extra=1360, airmass=1,
             model='invalid')
 
 
+def test_get_sky_diffuse_missing_dni_extra():
+    msg = 'dni_extra is required'
+    with pytest.raises(ValueError, match=msg):
+        irradiance.get_sky_diffuse(
+            30, 180, 0, 180, 1000, 1100, 100, airmass=1,
+            model='haydavies')
+
+
+def test_get_sky_diffuse_missing_airmass(irrad_data, ephem_data, dni_et):
+    # test assumes location is Tuscon, AZ
```

Comment:
```suggestion
    # test assumes location is Tucson, AZ
```

Reply:
https://en.wikipedia.org/wiki/Tucson,_Arizona

>The Spanish name of the city, Tucsón [tukˈson], is derived from the O'odham Cuk Ṣon [tʃʊk ʂɔːn], meaning "(at the) base of the black [hill]",

Reply:
wonder how many times I've mispelled 'Tucson' in the past....